deploy/gk-deploy

```diff
@@ -256,6 +256,9 @@ check() {
   if [[ ${cond} =~ $number_re ]]; then
     wait_limit=${cond}
   fi
+  if [[ "${resource}" == *s ]]; then
```

Comment:
Mainly concerned that this different semantics depending whether the first parameter ends with an 's' or not seems fragile and surprising to me, from the perspective of a user of this function.